src/program/lwaftr/nexthop/nexthop.lua

```diff
@@ -0,0 +1,68 @@
+module(..., package.seeall)
+
+local S = require("syscall")
+local ethernet = require("lib.protocol.ethernet")
+local lib = require("core.lib")
+local shm = require("core.shm")
+
+local ffi = require("ffi")
+local C = ffi.C
+
+local macaddress_t = ffi.typeof[[
+struct { uint8_t ether[6]; }
+]]
+
+local long_opts = {
+   help = "h"
+}
+
+local function usage (code)
+   print(require("program.lwaftr.nexthop.README_inc"))
+   main.exit(code)
+end
+
+-- TODO: Refactor to a general common purpose library.
+local function file_exists(path)
+   local stat = S.stat(path)
+   return stat and stat.isreg
+end
+
+local function parse_args (args)
+   local handlers = {}
+   function handlers.h (arg) usage(0) end
+   return lib.dogetopt(args, handlers, "h", long_opts)
+end
+
+local function is_current_process (pid)
+   return pid == S.getpid()
+end
+
+function run (args)
+   args = parse_args(args)
```

Comment:
args is never used, just call parse_args(args) here for the side effects, or assign it to _.